packages/shared/README.md

```
@@ -0,0 +1,98 @@
+
+// Use constants
+await setTimeout(ONE_SECOND);
+
+// Convert sizes
+const sizeInKB = bytesToKB(2048); // 2
+```
+
+## Integration Status
+
+✅ **@consentio/benchmark** - Fully integrated
+  - Uses shared constants for time and size conversions
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)